app/presenters/document/web_resource_presenter.rb

```diff
@@ -121,26 +153,28 @@ def technical_metadata
         runtime: render_document_show_field_value('ebucoreDuration'),
         runtime_unit: t('site.object.meta-label.runtime-unit-seconds'),
         attribution_plain: render_document_show_field_value('textAttributionSnippet'),
-        attribution_html:  render_document_show_field_value('htmlAttributionSnippet')
+        attribution_html: render_document_show_field_value('htmlAttributionSnippet')
       }
     end
 
     def is_avi?
-      avi_fmts = []
-      avi_fmts << 'video/avi'
-      avi_fmts << 'video/msvideo'
-      avi_fmts << 'video/x-msvideo'
-      avi_fmts << 'image/avi'
-      avi_fmts << 'video/xmpg2'
-      avi_fmts << 'application/x-troff-msvideo'
-      avi_fmts << 'audio/aiff'
-      avi_fmts << 'audio/avi'
-      avi_fmts.include? mime_type
+      %w(video/avi video/msvideo video/x-msvideo image/avi video/xmpg2
+         application/x-troff-msvideo audio/aiff audio/avi).include?(mime_type)
+    end
+
+    def displayable?
```

Comment:
Cyclomatic complexity for displayable? is too high. [11/6]<br>Assignment Branch Condition size for displayable? is too high. [18.87/15]<br>Perceived complexity for displayable? is too high. [11/7]